dll/cpl/sysdm/virtmem.c

```diff
@@ -569,27 +576,45 @@ OnSelChange(HWND hwndDlg, PVIRTMEM pVirtMem)
         }
 
         /* Set minimum pagefile size */
-        SetDlgItemText(hwndDlg, IDC_MINIMUM, _T("16 MB"));
+        SetDlgItemText(hwndDlg, IDC_MINIMUM, _T("2 MB"));
 
         /* Set recommended pagefile size */
         MemoryStatus.dwLength = sizeof(MEMORYSTATUSEX);
         if (GlobalMemoryStatusEx(&MemoryStatus))
         {
             FreeMemMb = (UINT)(MemoryStatus.ullTotalPhys / (1024 * 1024));
-            _stprintf(szBuffer, _T("%u MB"), FreeMemMb + (FreeMemMb / 2));
+            RecoMemMb =  FreeMemMb + (FreeMemMb / 2);
+            if(RecoMemMb>4096) RecoMemMb = 4096;
+            _stprintf(szBuffer, _T("%u MB"),RecoMemMb);
             SetDlgItemText(hwndDlg, IDC_RECOMMENDED, szBuffer);
         }
 
         /* Set current pagefile size */
-#if 0
         PageFileSizeMb = 0;
-        for (i = 0; i < 26; i++)
+        		
+        for (i = 0; i < pVirtMem->Count; i++)
         {
-            PageFileSizeMb += pVirtMem->Pagefile[i].InitialSize;
+		    _stprintf(szText,
+                          _T("%c:\\pagefile.sys"),
+                          pVirtMem->Pagefile[i].szDrive[0]);
+		
+		    hFind = FindFirstFileW(szText, &fdata);
+		    if (hFind == INVALID_HANDLE_VALUE) 
```

Comment:
again, mixed tabs and spaces...

Reply:
fixed... I have to change my notepad++ settings....

Reply:
fixed